make/crosscompile.jetson.mk

```diff
@@ -57,10 +57,10 @@ DEBUG = 0
 USE_SIGNAL_HANDLER = 1
 
 # the additional link flags you want to add
-ADD_LDFLAGS =
+ADD_LDFLAGS = -L${CROSS_ROOT}/lib
 
 # the additional compile flags you want to add
-ADD_CFLAGS =
+ADD_CFLAGS = -I${CROSS_ROOT}/include
```

Comment:
does this config get overwritten by users? or is it just included in makefile? for the former, should this change be included in makefile instead?

Reply:
Users might be able to add something to this file, but not override it. Otherwise, compilation simply failes.
I'd prefer to not pollute the Makefile with cross-compilation.